### DISCUSSION TITLE
Desktop Client (Flutter/Dart)

### DISCUSSION THREAD

**Discussion by trymeouteh** [breezy-weather/breezy-weather#1177] [Ideas]
### Describe your idea

A desktop client would be amazing, I would suggest if a desktop app is made it is written with Flutter/Dart which can also be used to compile the app for Windows, MacOS, Linux, Android and even iOS.

### Acknowledgements

- [X] I have searched the [existing issues](https://github.com/breezy-weather/breezy-weather/issues) and [existing discussions](https://github.com/breezy-weather/breezy-weather/discussions/categories/ideas) and this is a new ticket, **NOT** a duplicate or related to another open OR closed issue or discussion.
- [X] The idea I’m suggesting is not in the [list of features that will not be implemented](https://github.com/breezy-weather/breezy-weather/blob/main/README.md#features-that-will-not-be-implemented).
- [X] I have updated the app to **[version 5.2.5 or later](https://github.com/breezy-weather/breezy-weather/releases/latest)**. Even if I’m getting updates from F-Droid default repo, I don’t need to uninstall/reinstall to update.
- [X] I wrote my request in English, and any screenshot I attached are also in English (language can be changed from Settings > Appearance). I can write in my native language under the English text, if necessary.
- [X] I understand that this app is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY, so I may not get a positive reply to my request, if any at all.

**Comment by papjul** [comment]
You don't need Flutter to write desktop clients, Kotlin has multiplatform support.

Have a look at https://github.com/WangDaYeeeeee/GeometricWeather-Flutter for a Flutter client

**Reply by trymeouteh** [reply]
Can you compile a Kotlin app for Windows, MacOS and Linux and be able to develop the app as easily as developing a desktop app using Flutter?

**Reply by papjul** [reply]
I'm not a Flutter developer, feel free to try
